packages/create-schemas/src/bin.ts

```diff
@@ -29,3 +32,55 @@ function printConfigurationErrors(error: ZodError) {
     });
     console.log("Use --help to see available options.");
 }
+
+function printWatchEvents(initialConfig: ResolvedConfig, watcher: Watcher) {
+    let config = initialConfig;
+
+    printInputPath(config);
+
+    let start = Date.now();
+    watcher.on("start", () => {
+        start = Date.now();
+    });
+
+    watcher.on("change", () => {
+        console.log(blue("Change detected"));
+    });
+
+    watcher.on("configChanged", newConfig => {
+        console.log(blue("Configuration changed. Restarting."));
+        config = newConfig;
+        printInputPath(config);
+    });
+
+    watcher.on("error", error => {
+        if (error instanceof ZodError) {
+            printConfigurationErrors(error);
+        } else {
+            console.log(red("✖ Failed to generate schemas."));
+            console.log(red(String(error)));
+        }
+    });
+
+    watcher.on("done", content => {
+        const time = Date.now() - start;
+        console.log(`${green("✔")} Completed in ${time}ms`);
+        console.log(
+            `    ${bold(prettifyPath(fileURLToPath(config.output)))} ${dim(
+                `(${formatBytes(content.length)})`
+            )}`
+        );
+    });
+}
+
+function printInputPath(config: ResolvedConfig) {
```

Comment:
Also, we can only watch on 1 file right? given that we have a 1 BFF per app?

Reply:
Yep. We could technically watch on multiple files if we supported multiple inputs, but right now, the app assumes a single input.